extensions/control-plane/api/data-management/catalog-api/src/main/resources/ValidationMessages.properties

```diff
@@ -0,0 +1,14 @@
+#
+#  Copyright (c) 2020 - 2022 Microsoft Corporation
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Contributors:
+#       Microsoft Corporation - initial API and implementation
+#
+#
+catalog.get.providerUrl.NotNull.message=providerUrl must not be null
```

Comment:
Personally I would see no real problem in just hard-coding the message into the `@NotNull` annotation. In any event, we need to be consistent: either _all_ DTOs use the `*.properties` file (which should be done in a subsequent issue/pr), or _none_ do.

Reply:
I did hard-code the message at first, but then we would need to hard-code it in two places, in the interface and the impl
since the `@NotNull` should be equals in both places.

Reply:
Probably could be better to create a `ProviderDTO` with one field `providerUrl` and use this as first paramenter with `@Valid` annotation. wdyt?

Reply:
I've removed it and place the message in an hard-coded string